website/src/components/Sidebar.vue

```diff
@@ -16,32 +22,38 @@
 </template>
 
 <script>
+import SubLinks from '../subcomponents/SubLinks.vue'
 
 export default {
   props: {
     items: {
       type: Array,
       required: true
+    },
+    markdownHeadings: {
+      type: Array
     }
   },
 
-  computed: {
-    flatItems() {
-      const res = []
-      for (const item of this.items) {
-        res.push({ disabled: true, title: item.title })
-        for (const child of item.children) {
-          res.push({ ...child, disabled: false })
-        }
-      }
-      return res
+  components: {
+    SubLinks
+  },
+
+  data() {
+    return {
+      itms: []
     }
   },
 
+
   methods: {
     isActive(link) {
```

Comment:
should be a computed property

Reply:
doesn't work when I change it to a computed property